server/tests/unit/util.test.ts

```diff
@@ -32,6 +33,17 @@ describe(__filename(import.meta.url), () => {
     assert.strictEqual(result, true)
   })
 
+  it("Suppression des accents et caractères spéciaux des adresses emails", () => {
+    let cleanedEmail = cleanEmail("")
+    assert.strictEqual(cleanedEmail, "")
+
+    cleanedEmail = cleanEmail("àlan.léruŷêïÿt@test.fr")
+    assert.strictEqual(cleanedEmail, "alan.leruyeiyt@test.fr")
+
+    cleanedEmail = cleanEmail("jhönœ.dôœ.’£'^&/=!*?}ù@têst .com ")
+    assert.strictEqual(cleanedEmail, "jhono.doo.u@test.com")
```

Comment:
```suggestion
    assert.strictEqual(cleanEmail(""), "")
    assert.strictEqual(cleanEmail("àlan.léruŷêïÿt@test.fr"), "alan.leruyeiyt@test.fr")
    assert.strictEqual(cleanEmail("jhönœ.dôœ.’£'^&/=!*?}ù@têst .com "), "jhono.doo.u@test.com")
```
c'est plus facile à lire ;)

Reply:
Pour info, le code enlève aussi les accents des majuscules